blockstore/splitstore/README.md

```diff
@@ -59,6 +59,17 @@ These are options in the `[Chainstore.Splitstore]` section of the configuration:
   nodes beyond 4 finalities, while running with the discard coldstore option.
   It is also useful for miners who accept deals and need to lookback messages beyond
   the 4 finalities, which would otherwise hit the coldstore.
+- `HotStoreMovingGCFrequency` -- specifies how frequenty to garbage collect the hotstore
```

Comment:
yup, will rename to `FullGCFrequency`.

Reply:
IMO "moving" is already the most pessimistic/common-denominator form of GC. I do think it generalises nicely because it's the ultimate form of "GC" that databases that do not support efficient GC will fall back to. It's basically a backstop/catch-all that is applicable to all and any database.

Reply:
That's a good point, but I think that steven's "FullGC" nomenclature is right -- the user shouldn't care about the mechanics, as long as it really is a full gc; besides I have already renamed it :)
(but I can rename back if we deem so)

Reply:
done.